scrapper.py

```diff
@@ -1,10 +1,143 @@
 import requests
 from bs4 import BeautifulSoup
+from sys import argv
+import re
 
-page = requests.get('http://127.0.0.1:5500/krisha_scrapper/flat_list.html')
-soup = BeautifulSoup(page.content, 'html.parser')
+def check_cli_arg(command_line_arguments: list) -> str:
+    """ Function checks:
+            - that URL was entered on command line, if not promts entering URL
+            - URL starts with 'https://krisha.kz/', if not promts entering correct URL or exit
+            and return URL
+    """
 
-prices = soup.select("html body div.cnt-aprts-lst div.apartment div.price")
-first5 = prices[:5]
-for price in first5:
-    print(price.text)
\ No newline at end of file
+    if len(command_line_arguments) == 1:
+        cli_argument = input("Don't input URL. Input URL:")
```

Comment:
@ProvVlad , I assume the message to a user should be "You missed an argument. It is expected a URL that should be grabbed."